### PR TITLE
Generated dialogs documentation

### PR DESCRIPTION
**Pull request by clearab**
Create a scaffold and initial set of documentation for the generated dialogs plugin.

Goals:

- Consolidate documentation to the top-level README
- Minimize duplication in the lower-level readme files, redirect to all-up documentation
- Standardize the plugin documentation to align to that in the botframework-cli repo

Fixes https://github.com/microsoft/botframework-sdk/issues/5734

### CODE REVIEW

File: experimental/generation/readme.md [hunk 1]
```diff
@@ -1,32 +1,112 @@
...
-- Support for choosing between ambiguous entity values and entity property mappings.
-- Recognizing and mapping all LUIS prebuilt entities.
-- Help including auto-help on multiple retries.
+The Bot Framework has a rich collection of conversational building blocks, but creating a bot that feels natural to converse with requires understanding and coordinating across language understanding, language generation and dialog management.
+To simplify this process and capture best practices, we've created the [bf-generate][bf-generate] plugin for the [BotFramework CLI tool][bf].
+The generated dialogs make use of event-driven adaptive dialogs with a rich and evolving set of capabilities including:
+
+- Handle out of order and multiple responses for simple and array properties.
+- Add, remove, clear and show properties.
+- Response ambiguity with entity values and property mappings.
```
Comment (chrimc62) [inline experimental/generation/readme.md:9]: >Response ambiguity with entity values and property mappings. [](start = 2, length = 60)

This seems less clear to me.

File: experimental/generation/generator/README.md [hunk 2]
```diff
@@ -9,220 +9,62 @@ Generate Bot Framework Adaptive Dialogs from JSON schema.
...
+  bf dialog:generate -s {SCHEMA FILE} -o {OUTPUT LOCATION}
+
+```
+
+_See [source code](src/commands/generate.ts).
+
+<!-->
+## `bf dialog:integrate`
+
+asdf
```
Comment (chrimc62) [inline experimental/generation/generator/README.md:52]: ???

File: experimental/generation/generator/README.md [hunk 3]
```diff
@@ -9,220 +9,62 @@ Generate Bot Framework Adaptive Dialogs from JSON schema.
...
+
+```
+USAGE
+
+asdf
+
+OPTIONS
+
+adfs
```
Comment (chrimc62) [inline experimental/generation/generator/README.md:61]: ???

File: experimental/generation/generator/README.md [hunk 4]
```diff
@@ -9,220 +9,62 @@ Generate Bot Framework Adaptive Dialogs from JSON schema.
...
+
+EXAMPLES
+
+asdf
```
Comment (chrimc62) [inline experimental/generation/generator/README.md:65]: ???

File: experimental/generation/runbot/readme.md [hunk 5]
```diff
@@ -12,13 +16,17 @@ In order to setup runbot:
...
-In order to use runbot, you execute the command `dotnet run <pathToRepo>/experimental/generation/runbot/runbot.csproj --root <directoryWithDeclarativeAssets>`. At that point you can connect to your bot using `http://localhost:5000/api/messages` in the [Bot Framework Emulator][emulator].
+
+In order to use runbot, you execute the command `dotnet run <pathToRepo>/experimental/generation/runbot/runbot.csproj --root <directoryWithDeclarativeAssets>`.
+At that point you can connect to your bot using `http://localhost:5000/api/messages` in the [Bot Framework Emulator][emulator].
 
 Command line args:
+
 * **--root <PATH>**: Absolute path to the root directory for declarative resources all *.main.dialog be options.  Default current directory");
 * **--region <REGION>**: LUIS endpoint region.  Default westus");
-* **--environment <ENVIRONMENT>**: LUIS environment settings to use.  Default is user alias.  This is used to find your `luis.settings.<environment>.<region>.json` settings file for luis.
```
Comment (chrimc62) [inline experimental/generation/runbot/readme.md:20]: >Default is user alias.  This is used to find your `luis.settings.<environment>.<region>.json` settings file for luis. [](start = 70, length = 117)

Why did you remove this?

File: experimental/generation/generator/README.md (hunk 2 above)
Comment (clearab) [reply experimental/generation/generator/README.md:52]: This entire section was supposed to be commented out - I wasn't sure if you wanted to document the command as-is, or wait for the integrated --integrate switch?

**Comment by chrimc62** [comment]
I have a PR with –merge as the switch and some documentation.  I’m working with Zongyang to fix some bugs though before it all gets integrated.

File: experimental/generation/runbot/readme.md (hunk 5 above)
Comment (clearab) [reply experimental/generation/runbot/readme.md:20]: It's still there, just the line endings formatting change

File: experimental/generation/readme.md (hunk 1 above)
Comment (clearab) [reply experimental/generation/readme.md:9]: updated to previous verbiage

File: experimental/generation/generator/README.md (hunk 2 above)
Comment (clearab) [reply experimental/generation/generator/README.md:52]: removed
